packages/data/src/world/mm/overworld/southern_swamp.yml

```diff
@@ -136,7 +143,7 @@
     PICTURE_SWAMP: "has(PICTOGRAPH_BOX)"
     PICTURE_BIG_OCTO: "is_swamp_poisoned && has(PICTOGRAPH_BOX) && soul_octorok"
```

Comment:
We can add the left Octo event here too. Bow can be done easily, hook if using hovers. Dont know about dins or short hook with hovrrs.